### ISSUE TITLE
X-linked Strictly Recessive inheritance

### ISSUE THREAD

**Issue by angharadroberts** [obophenotype/human-phenotype-ontology#7488]
**Preferred term label:**
X-linked Strictly Recessive inheritance

**Synonyms**

**Definition (free text, please give PubMed ID)**
X-linked recessive inheritance that is strictly recessive and in which females do not manifest a mild disease phenotype

**Parent term (use hpo.jax.org/app)**
X-linked recessive inheritance HP:0001419

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@angharadroberts 

Note I have modified the definition to remove the word "mild" (which would leave open the possibility that females manifest a severe phenotype).

The new term has the id HP:0034337

X-linked recessive inheritance that is strictly recessive and in which females do not manifest a disease phenotype.

**Comment by pnrobinson** [comment]
Actually, here is another attempt at a definition to provide more clarity and also to cover the edge case of a female who has inherited one mutation and has a de novo mutation in a gene associated with a strictly recessive disease:

X-linked recessive inheritance that is strictly recessive and in which females (with a heterozygous genotype) do not manifest a disease phenotype.

**Comment by kanems** [comment]
Was there a determination not to add this and the similar term of "X-linked primarily recessive" inheritance?
I was just spot checking some of these to make sure MedGen processing hadn't missed them or incorrectly categorized them and I see [HP:0034337](https://hpo.jax.org/app/browse/term/HP:0034337) is "claw hand deformity" not this inheritance concept, nor are either term children of the parent as suggested in this ticket.
